train_linear_model.py

```diff
@@ -0,0 +1,35 @@
+import argparse
+import pandas as pd
+import joblib
+from sklearn import linear_model
+from sklearn.metrics import mean_squared_error, r2_score
+
+from linear_model import X_test
+
+
+def train(X_train, y_train, saved_model):
```

Comment:
same comment re: naming of variable, and yea, probably better to load the features dataframe and create `X_train`, `y_train` on the spot